src/app/system/navigation/navigate.ts

```diff
@@ -157,7 +157,7 @@ export async function navigate(url: string, options: NavigateOptions = {}) {
         requestAnimationFrame(pushView)
       }, timeoutTime)
     } else {
-      pushView()
+      requestAnimationFrame(pushView)
```

Comment:
does this need to be like this? I guess not since it works 🤷 
```suggestion
      requestAnimationFrame(pushView())
```

Reply:
My favorite robot says (paraphrased) that `requestNavigationFrame(pushView())` will execute pushView immediately and pass the result to requestNavigationFrame, whereas `requestNavigationFrame(pushView)` passes the function reference and gives requestNavigationFrame control of when to execute pushView, which is what we want in this case. I didn't know that before though I just copied it from the similar call above so thanks for asking!